docs/source/api_reference/lpa_utilities.rst

```diff
@@ -21,7 +21,9 @@ Beam utilities
 
 .. autofunction:: fbpic.lpa_utils.bunch.add_elec_bunch_gaussian
 
```

Comment:
This is in order to have this function added in the FBPIC automatic documentation.